### DISCUSSION TITLE
help this gyt bgodmaire made this account and has been stalking me idk what this is but he has all my info plz help

### DISCUSSION THREAD

**Discussion by bgodmare** [community/community#127007] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

these ppl are sick they poisoned my dog,,I found this on my CPU doing some research

**Comment by ebndev** [comment]
I’m sorry that you’re having to go through this, @bgodmare. 

It sounds like you have been a victim of serious harassment and/or abuse. Though we can’t directly help with your situation, you might be able to receive assistance through [Crash Override Network](http://www.crashoverridenetwork.com/) and [RAINN](https://www.rainn.org/).
